### DISCUSSION TITLE
List of Events

### DISCUSSION THREAD

**Discussion by delbertina** [Zrips/CMI#2550] [Q&A]
**Question: Is there a list of valid events for the 'eventCommands' config file?**

Noticed there's an event in the default file for kick but not ban, mute etc. The custom event list from the wiki page on the API has a different naming scheme so am unsure if that's the same thing. Sorry if this is a noob question. :)

**Comment by CloudeLecaw** [comment]
you have the eventCommand file where all events listed

**Comment by mrfloris** [comment]
The events supported are in that file, adding custom ones isn't possible. If there's any missing feel free to suggest them for consideration. What happens with mute for example is handled in config.yml (as to what they can do, etc) I can't think of a reason mute should be added, but ban might be helpful. 

Any others you're missing specifically?

**Comment by mrfloris** [comment]
`Added playerBan into eventCommand file`
CMI 9.0.3.x
